docs/second_factor_auth.md

```diff
@@ -0,0 +1,54 @@
+# Second Factor Authentication (2FA)
+
+According with [wikipedia](https://en.wikipedia.org/wiki/Multi-factor_authentication) Two-factor authentication (also known as 2FA) is a type (subset) of multi-factor authentication. It is a method of confirming a user's claimed identity by utilizing a combination of two different factors: 1) something they know, 2) something they have, or 3) something they are.
+
+This feature provider 2FA for OpenStack Keystone based on email.
+
+## Configuration
+
+Specific options at /etc/keystone/keystone.conf
+```
+[spassword]
+sndfa=true
+sndfa_time_window=24
+sndfa_endpoint='localhost:5001'
+```
+
+`sndfa` is a boolean which enables (true) or disables (false) if Second Factor Authentication feature is available in Keystone instance.
```

Comment:
Formatting suggestion: use a bullet list for this.

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/57/commits/20ef18a09749b0702a2a88d40475194fb490d370